packages/php-wasm/web-service-worker/src/initialize-service-worker.ts

```diff
@@ -64,7 +64,8 @@ async function defaultRequestHandler(event: FetchEvent) {
 			// cookies
 			credentials: 'omit',
 		});
-		return fetch(request);
+		// TODO: Is this necessary? Do we really want to bypass browser cache for all requests?
+		return fetchFresh(request);
```

Comment:
It depends, this is a technically simple solution that will work, but it removes all the benefits of browser caching.

My problem with this is that the client takes over control of cache busting and we don't control a running client. 

I would prefer if we kept cache headers and focused on ensuring the version hash generated by builds works for all critical resources. 

Once this is working we can fetch by default. If the version is the same, the browser will return a cached resource.

If the connection is unstable, or offline, we let the service worker return the resource from cache, but otherwise, we don't rely on the service worker's cache.

Reply:
> I would prefer if we kept cache headers and focused on ensuring the version hash generated by builds works for all critical resources.

We can't have a version hash for at least a few routes: `/sw.js`, `/index.html`, `/remote.html`, `/`, and probably some more like perhaps `/manifest.json`.